x-pack/packages/security-solution/features/src/security/product_feature_config.ts

```diff
@@ -79,10 +79,17 @@ export const securityDefaultProductFeaturesConfig: DefaultSecurityProductFeature
 
   [ProductFeatureSecurityKey.endpointArtifactManagement]: {
     subFeatureIds: [
+      SecuritySubFeatureId.hostIsolationExceptions,
```

Comment:
NIT: Maybe it would be worth renaming this ID to `SecuritySubFeatureId.hostIsolationExceptionsBasic` since it's included in the `endpointArtifactManagement` PLI (essentials) and only gives _read_ and _delete_ privileges.